nixos/modules/testing/test-instrumentation.nix

```diff
@@ -226,6 +226,11 @@ in
 
     # Squelch warning about unset system.stateVersion
     system.stateVersion = lib.mkDefault lib.trivial.release;
+
+    # Don't pull in switch-to-configuration. This is mostly a Hydra optimization,
+    # so we don't rebuild all the tests every time switch-to-configuration-ng changes.
+    system.switch.enable = lib.mkDefault false;
+    system.switch.enableNg = lib.mkDefault false;
```

Comment:
Don’t set `enableNg`, just `enable` is enough to get it out of the closure. Disabling `enableNg` might result in tests that opt‐in getting the old `switch-to-configuration` instead.